src/scss/_modal-overlay.scss

```diff
@@ -27,14 +26,21 @@
  * Block clicks except for those that would occur
  * on Shepherd elements or on the target element.
 */
-.shepherd-active.shepherd-modal-is-visible :not(.shepherd-target) {
-  pointer-events: none;
-}
+.shepherd-active {
+  &.shepherd-modal-is-visible {
+    :not(.shepherd-target) {
```

Comment:
Yeah, so it looks like without the `*`, this rule actually take precedent on _all_ elements inside of it.